components/Buttons/MainButton/MainButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Link from "next/link";
+
+import ButtonBase from "@material-ui/core/ButtonBase";
+import Typography from "@material-ui/core/Typography";
+
+import { useStyles } from "./styles";
+
+type props = {
```

Comment:
Use PascalCase for types names

https://basarat.gitbook.io/typescript/styleguide#type